adapters/handlers/rest/configure_api.go

```diff
@@ -506,13 +506,15 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 			Fatal("modules didn't initialize")
 	}
 
+	storeReadyCtx, storeReadyCancel := context.WithCancel(context.Background())
```

Comment:
should we add `defer storeReadyCancel()` just to be sure? (say something crashes inside following `GoWrapper` and leaving waiting on line 599: `<-storeReadyCtx.Done()` hanging?). 

Plus, calling cancel after it got cancelled is no-op anyway.

Reply:
actually I can't because cancel is called from goroutine, which can happen after main function is already finished. `defer cancel()` will be called before meta store is ready.